paradise_papers_search/fetch_api/models/Address.py

```diff
@@ -22,5 +24,16 @@ def serialize(self):
                 'node_id': self.node_id,
             },
         }
-
+    @property
+    def serialize_connections(self):
+        return [
+            {
+                'nodes_type': 'officers',
+                'nodes_related': helpers.serialize_relationships(self.officers.all(), 'REGISTERED_ADDRESS'),
+            },
+            {
+                'nodes_type': 'intermediaries',
```

Comment:
We re expecting the nodes type value in singular, for consistency reasons: `intermediary` and `officer`